lib/cb/requests/job_search/get.rb

```diff
@@ -29,7 +29,7 @@ def http_method
 
         def headers
           {
-            'HostSite' => args[:host_site] || Cb.configuration.host_site,
```

Comment:
Was anybody using this? Is there any way to tell (I mean, it didn't do anything, right?)